transcom/mymove
# B:18830 - Add Office Account Request Form

## [B-18830](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18830)
## MERGE AFTER [B-19465](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19465) - Currently in Integration

## Summary

This MR introduces functionality that will allow a user to request an office account by filling out a form.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Office Sign In Page
2. Select the Request Account tab in the top right corner of the navigation bar
3. Fill out the required information on the form then select Request Account Button
4. Check database table office_users. You should see the record with a REQUESTED status. You can also check the network tab for a 200.

### Frontend


## Screenshots
